subsys/net/ip/net_context.c

```diff
@@ -229,7 +229,8 @@ int net_context_get(sa_family_t family,
 			continue;
```

Comment:
Please remove the "subsys:" from commit subject, so it should start with "net: offload: ...." in this case.

Reply:
done